### ISSUE TITLE
[Question] What's the future of this plugin past IJ 2020.3 release?

### ISSUE THREAD

**Issue by ChrisCarini** [weisJ/auto-dark-mode#23]
I was reading JetBrain's blog post today on [IntelliJ IDEA 2020.3 EAP](https://blog.jetbrains.com/idea/2020/09/intellij-idea-2020-3-eap/) which covers some of the features coming to the next release.

One of the features highlighted in this post is `IDE theme synced with OS settings` which seems to natively bring to IJ the feature this plugin brings.

I was just curious what this means for the `Auto Dark Mode` plugin. Will this be kept around to continue to bring the feature / functionality to older versions of IJ / other IDEs?

**Comment by weisJ** [comment]
Good question.
The plugin will continue to receive updates for future versions of IntelliJ. At least as long our features exceed those of IntelliJ itself (e.g. IntelliJ still doesn’t support Gnome, but that may change until the release of IJ2020.3).

I want to note that the implementation in IntelliJ itself is inherently the same as the one this plugin uses and there shouldn’t be any performance differences between the two implementations.

Also if there are any necessary bug fixes or new features are added (e.g. support for more Linux environments) those will be backported to older versions of the IntelliJ-Platform. The oldest version which will receive updates will be based on the version AndroidStudio uses, as it’s the only one lacking behind with the IJ-Platform version.

**Comment by insign** [comment]
I'm very thankful for this plugin since I tried to fix this in this new EAP, and this issue showed me how to. Thanks.

**Comment by weisJ** [comment]
Update: Android studio has been updated to be based on the 2020.1 version. Thus version `1.5.4-2019.3` will be the last version of this plugin published for `IDEA 2019.3` based IDEs.

**Comment by takanuva15** [comment]
@weisJ Hi, I was wondering if you could add a message to the top of the README stating that IJ now supports this feature natively so the plugin is no longer necessary. (I just updated to IJ 2021 today and I got worried when I saw that this plugin had no 2021 version out.)

**Comment by weisJ** [comment]
Sure thing :)

**Comment by atl-mk** [comment]
Thanks for building this, I'm still using it because of the Linux support

**Comment by steebchen** [comment]
I'm not sure but for some reason the native IJ feature doesn't work, it often uses the wrong IDE theme, and NEVER uses the correct respective light/dark syntax theme for me... so personally I definitely need this plugin and I'm happy it exists.

**Comment by victor-falcon** [comment]
Yes please, keep supporting new IDE versions because the IDE options change the theme, but you can specify which color schema you want for each light or dark themes.
